External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -1344,31 +1344,94 @@ static private void ShowCalltip(ScintillaControl Sci, int paramIndex, bool force
             else UITools.CallTip.CallTipSetHlt(start + 1, end, true);
         }
 
-        static string[] featStart = new string[] { "/*", "{", "<", "[", "(" };
-        static string[] featEnd = new string[] { "*/", "}", ">", "]", ")" };
-
         static private int FindNearSymbolInFunctDef(string defBody, string symbol, int startAt)
```

Comment:
There are /* and */, so I thought that maybe in the future there may be needs for other strings and decided to make it more generic... but probably /* and */ could be treated separatedly and only care about chars.